content/doc/developer/plugin-development/index.adoc

```diff
@@ -2,30 +2,11 @@
 title: Plugin development
 summary: About Jenkins' internal extension functionality, modules, and plugins
 layout: developerchapter
-wip: true
 references:
 - url: https://github.com/jenkinsci/plugin-pom
   title: Plugin parent POM on GitHub
-- url: https://wiki.jenkins.io/display/JENKINS/Adding+tool+auto-installer
```

Comment:
does this need adding? It doesn't seem recommended to implement these any more? From what I've seen it's either
1. cache it on your server with a URL based tool
2. add it to your agent image

Reply:
Definitely not a priority. CC @daniel-beck .

Reply:
Just keep the link in if it's not getting migrated?